styles/globals.scss

```diff
@@ -94,7 +94,7 @@ body {
 
 .notifications {
     z-index: 100;
-    position: absolute;
+    position: fixed;
```

Comment:
why this update?

Reply:
The notification won't follow if I scroll down, it keeps itself at the same position. If i scroll down it should follow me imo.